suite-native/notifications/src/components/TransactionNotificationDescription.tsx

```diff
@@ -17,24 +15,16 @@ const addressContainerStyle = prepareNativeStyle(_ => ({
 }));
 
 export const TransactionNotificationDescription = ({
-    amount,
+    formattedAmount,
     prefix,
     targetAddress,
-    networkSymbol,
 }: TransactionNotificationDescriptionProps) => {
     const { applyStyle } = useNativeStyles();
 
     return (
         <Box flexDirection="row">
-            <CryptoAmountFormatter
```

Comment:
Never even use `formattedAmount`. That is nonsense from desktop Suite which is bad for many reasons. Always pass raw value and only format it using formatter. Here you should use `amount` field from `Transaction`.